emcc.py

```diff
@@ -2338,6 +2338,8 @@ def phase_linker_setup(options, state, newargs):
         '_wasmfs_read',
         '_wasmfs_pread',
         '_wasmfs_symlink',
+        '__syscall_truncate64',
+        '__syscall_ftruncate64',
```

Comment:
Was there a reason to use the raw syscalls and not a `_wasmfs_*` wrapper? Wrappers have the benefit that changes to the syscalls don't affect them.

Reply:
Thomas suggested a little earlier that since `truncate` doesn't have any additional processing, it might make sense to directly use the raw syscall. However, I do see your point that a wrapper would be beneficial.

Reply:
I think for now you should continue to use wrapper functions and we can separately/later consider weather they are actually needed since (IIUC) these are not the only once that are simple pass-through functions?

Reply:
Don't block landing anything on this conversation, but I still think pass-through wrappers aren't that useful. If we update the syscalls (which should be extremely rare), then we would still have to update their callers. Whether those callers are in C or JS doesn't make much of a difference as far as I can tell.